### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by HichamJohra**
I have tried to make a more detailed explanation of what the DESTEST is all about

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
 # IBPSA Project 1 - District Energy Simulation Test Procedure (DESTEST)
 
-This repository contains the DESTEST developed within the IBPSA Project 1.
+To establish reliable, decarbonized and sustainable cities, it is necessary to integrate all energy grids together with the different energy suppliers and end-users during the design, optimization and operation phases. To that matter, accurate urban-scale numerical models have to be developed to dynamically simulate buildings, infrastructures, networks, energy and matter fluxes. The multi-physics modelling language Modelica is particularly well-suited for that purpose.
+
+The goal of the IBPSA Project 1 is to develop a BIM/GIS and Modelica framework for building and community energy simulation. Within the IBPSA Project 1, the DESTEST (Work Package 3.1) aims at testing district energy system simulations tools, and in particular, validate the urban energy systems models of the dedicated Modelica libraries.
```
Comment (MichaMans) [inline README.md:5]: Here I would add the links to the libraries and add a sentence like: The dedicated Modelica libraries can be found here: and they are based on the modelica-ibpsa (link) core library.

**Review by MichaMans — CHANGES_REQUESTED** [review]
@HichamJohra Thanks! This is a much better description then before 😃 I would suggest to integrate the links to the Modelica libraries and then merge it

**Comment by HichamJohra** [comment]
@MichaMans ok perfect.
Can you send me the links to the different Modelica libraries and the core one? Just to make sure I don't forget any and that I place the right link.

thanks

**Comment by MichaMans** [comment]
@HichamJohra Here ist the core library with the links to the end-user libraries in the readme. https://github.com/ibpsa/modelica-ibpsa

**Pull request merged**